Duplicati/Library/Backend/Telegram/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net471" />
```

Comment:
I'm seeing build warnings for conflicting versions of Newtonsoft.Json:
```
Sep 20 15:15:40 /usr/lib/mono/msbuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2101,5): warning MSB3277: Found conflicts between different versions of "Newtonsoft.Json" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [/application/Duplicati/Library/Backend/Telegram/Duplicati.Library.Backend.Telegram.csproj]
```
The other projects in the solution require version 12.0.0.0.  Can we do the same here?